public/app/features/alerting/unified/components/rule-editor/query-and-alert-condition/QueryAndExpressionsStep.tsx

```diff
@@ -241,7 +253,11 @@ export const QueryAndExpressionsStep = ({ editingExistingRule, onDataChange }: P
       // As a workaround we update form values as soon as possible to avoid stale state
       // This way we can access up to date queries in runQueriesPreview without waiting for re-render
       const previousQueries = getValues('queries');
-      const expressionQueries = previousQueries.filter((query) => isExpressionQuery(query.model));
+
+      // @ts-ignore
```

Comment:
Why we need to ignore ts?

Reply:
TS is not able to recognize that the result will be ` AlertQuery<ExpressionQuery[]`